src/ApiCalendar.js

```diff
@@ -239,15 +241,17 @@ class ApiCalendar {
      * Update Calendar event
      * @param {string} calendarId for the event.
      * @param {string} eventId of the event.
+     * @param {string} sendUpdates Acceptable values are: "all", "externalOnly", "none"
      * @param {object} event with details to update, e.g. summary
      * @returns {any}
      */
-    updateEvent(event, eventId, calendarId = this.calendar) {
+    updateEvent(event, eventId, sendUpdates = false, calendarId = this.calendar) {
```

Comment:
Pass the argument as last argument to avoid breaking change please

Reply:
You are saying sendUpdates can be equal to "all", "externalOnly" or "none" but you are making it equal to false by default.